bugbot/utils.py

```diff
@@ -765,3 +765,28 @@ def is_keywords_removed_by_bugbot(bug: dict, keywords: Iterable) -> bool:
         if change["field_name"] == "keywords"
         for keyword in keywords
     )
+
+
+def get_bug_bugdash_url(component: str, triage=False) -> str:
```

Comment:
We could accept it as `ComponentName` instead of `str`? We then could form the name in the way that we want instead of haking it with `replace()`:
```py
encoded_component = quote_plus(f"{component.product}:{component.name}")